app/brave_settings_strings.grdp

```diff
@@ -1550,6 +1550,12 @@
   <message name="IDS_SETTINGS_LEO_ASSISTANT_SHOW_IN_CONTEXT_MENU_DESC" desc="The description for settings option to show Leo in context menu">
     Leo will appear in the context menu when you right-click on a website. This will send context from the current web page to Leo.
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_AUTOMATICALLY_ATTACH_PAGE_LABEL" desc="The text for settings option to automatically attach page to Leo">
+    Automatically add content from the active tab for context
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_AUTOMATICALLY_ATTACH_PAGE_DESC" desc="The description for settings option to automatically attach page to Leo">
+    Toggle this on for new conversations started from the sidebar to be contextually aware of the page you launched them from. The contents will be sent to Brave Leo along with your messages.
```

Comment:
i'm not sure users will understand what "contextually aware" means or what "the contents" refer to. maybe we can reuse some copy from https://github.com/brave/reviews/issues/1946#issuecomment-2880974856 as it would be good to be consistent here. also we should follow the grammatical construct of the other toggles, AKA the subtext simply says what happens when the toggle is on.

like `When starting a new Leo conversation from the sidebar, Brave will automatically attach the active page's content to give Leo more context. The page content will be sent with your message to Leo unless you remove it.`

Reply:
Sweet, that's a much better message. I've updated it :smile:

Also, trying to use `Page` over `Tab` :laughing: